buildenv/jenkins/common/variables-functions

```diff
@@ -428,11 +428,11 @@ def set_slack_channel() {
 }
 
 def set_artifactory_config() {
-    ARTIFACTORY_SERVER = VARIABLES.artifactory_server
-    ARTIFACTORY_REPO = VARIABLES.artifactory_repo
+    env.ARTIFACTORY_SERVER = VARIABLES.artifactory_server
+    env.ARTIFACTORY_REPO = VARIABLES.artifactory_repo
```

Comment:
Because these are set in the Build job, in order to sniff it in the upstream pipeline it has to be written to the environment. The only one we _need_ for this change is the server but the test code will accept repo, num artifacts as well. See PR in description.